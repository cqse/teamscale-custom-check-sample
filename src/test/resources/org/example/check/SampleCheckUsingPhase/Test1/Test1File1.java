package org.example.check.SampleCheckUsingPhase.Test1;

public class SampleCheckCode {
	public class Inner {

	}
}
