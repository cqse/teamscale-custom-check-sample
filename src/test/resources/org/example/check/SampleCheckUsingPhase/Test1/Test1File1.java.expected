Offsets 88-111 (lines 4-6): found another type with same name